writer/writer/core/writer_service.py

```diff
@@ -25,37 +26,48 @@ class WriterService:
     messaging: Messaging
 
     def write(
-        self, write_request: WriteRequest, log_all_modified_fields: bool = True
+        self,
+        write_requests: Union[WriteRequest, List[WriteRequest]],
+        log_all_modified_fields: bool = True,
     ) -> None:
-        with self._lock:
-            self.write_request = write_request
+        if isinstance(write_requests, list):
```

Comment:
Please move this to the HTTP-layer: Let the writer service just accept a list of write requests.

Reply:
In backend-terminology it's not a list of WriteRequests, it's a list of Actions. Each action consists of a list of WriteRequest and a list of locked_fields.

Reply:
ok